### ISSUE TITLE
[Bug] Your requirements could not be resolved

### ISSUE THREAD

**Issue by ugintl** [singularo/stratoserp_base#8]
I am on a lamp server. I am getting following error.

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested PHP extension ext-posix * is missing from your system. Install or enable PHP's posix extension
```

Is it also required if I just installing the module?

**Comment by singularo** [comment]
I'm unlikely to test or support with lamp sorry. You might be able to just comment that as its likely related to the docker/robo parts. Will consider merging changes if you can confirm whats needed.

**Comment by ugintl** [comment]
The `composer create` command was not working for me. It was asking for ext-posix. Also tried to remove ext-posix requirement, but It was still asking for it. stratoserp, stratoserp_base as well as drupal-shepherd-scaffold required ext-posix. It is because in the beginning of the composer.json file in the site root directory, it has links to the these repositories and it is ignoring the local composer file. I changed the version of drupal shepherd scaffold from dev-develop to 4.0, but it said it needs composer 1. 
 
What I did then?

I downloaded the repository, ran `composer update` in it. It created "web" and "vendor" folders within the repository. Then I moved the contents of install profile under profiles/contrib/stratoserp_base. Voila, it installed correctly, but obviously, it installed everything in the install profile repository including the composer.lock file. I know it is not the right way. There is no composer file in the site root directory outside "web" folder. I am still trying to find out how to do it the right way and at the same time avoid ext-posix requirement.

Are you getting the picture?
